crates/fuel-streams-protobuf/proto/fuel_types.proto

```diff
@@ -0,0 +1,20 @@
+
+syntax = "proto3";
+
+package fueltypes;
+
+message Block {
+    BlockHeader header = 1;
+    repeated Transaction transactions = 2;
+}
+
+message BlockHeader {
+    uint32 da_height = 1;
+    uint32 consensus_parameters_version = 2;
+    uint32 state_transition_bytecode_version = 3;
+}
+
+message Transaction {
+    bytes mint_asset_id = 1;
+    uint64 mint_amount = 2;
+}
```

Comment:
I am just making a note here: This structure is in reality very nested with a lot of variants for Mint, Script etc. - highly unpractical for mapping to protobuf as the latter does not support what Rust does.